elixir-mode.el

```diff
@@ -330,6 +305,33 @@ for the Elixir programming language."
     (,(elixir-rx (group sigils))
      1 font-lock-builtin-face)
 
+    ;; Sigil patterns. Elixir has support for eight different sigil delimiters.
```

Comment:
It looks like you modified this correctly *but* you left [these lines](https://github.com/rranelli/emacs-elixir/blob/highlight-all-modules-equally/elixir-mode.el#L335-L360) in tact. Please remove them.